cmd/agent/agent.go

```diff
@@ -249,9 +249,11 @@ func startManager(o *options.AgentOptions, ctx context.Context) {
 			os.Exit(1)
 		}
 
-		if err = clusterClaimReconciler.SetupWithManager(mgr); err != nil {
-			setupLog.Error(err, "unable to create controller", "controller", "ClusterClaim")
-			os.Exit(1)
+		if o.EnableSyncLabelsToClusterClaims == true {
```

Comment:
I am thinking if it is better to make the SyncLabelsToClusterClaims enable as default, and only disable it for the sd hub? if yes, this could be `o.DisableSyncLabelsToClusterClaims` ?

Reply:
It is enabled by default now.